packages/app-mobile/utils/fs-driver-rn.ts

```diff
@@ -36,6 +36,14 @@ export default class FsDriverRN extends FsDriverBase {
 		if (isScopedUri(path)) {
 			return RNSAF.writeFile(path, content, { encoding: encoding as Encoding });
 		}
+
+		// Work around a bug in rn-fetch-blob on Android: Unicode
+		// characters were written as ??s.
+		if (encoding === 'utf-8' || encoding === 'utf8') {
```

Comment:
Just a general comment, as I'm not familiar with the code base or TypeScript. Would the encoding parameter always be passed in as a lowercase value, or might this check need to be case insensitive?

Reply:
I'll have to do tests with this, but it might make sense to pass the encoding directly to the [`Buffer` constructor](https://github.com/feross/buffer/blob/50cae94c3a683e43aa679c7d0add058b443dfee1/index.js#L927) instead.

This would provide (more) compatibility with the Node `fs-driver`'s `writeFile` implementation, and should also check that `encoding` is valid.